uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -52,10 +52,14 @@ define([
           public_holiday: isPublicHoliday || false
         };
 
-        deferred.resolve(this.sendGET('LeaveRequest', 'getbalancechangebyabsencetype', params, false)
+        this.sendGET('LeaveRequest', 'getbalancechangebyabsencetype', params)
```

Comment:
You are reverting the `false` parameter, used to disabled caching on the ajax call (introduced [here](https://github.com/civicrm/civihr/pull/1297/commits/92c64494ae2bb1223fed05ce22de6a9b52edb468#diff-2580b71d3d2a79bd0a3154318a126cb7R58))